Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -16,23 +16,18 @@
 namespace Stratis.Bitcoin.Features.Consensus
 {
     /// <summary>
-    /// A block object that is passed around when downloading a block from peers and has information about the validation state of the block.
+    /// Information about a block that is required for its validation. 
+    /// It is used when a new block is downloaded or mined.
     /// </summary>
-    public class BlockItem
+    public class BlockValidationContext
```

Comment:
This is a good name yes, come to think about it I would actually swap this with the validation context so  `ContextInformation` to `ValidationContext` and this field to be higher level perhaps `ConsensusContext`.

Leave it as is for now.